domain/generator/src/main/kotlin/com/few/generator/usecase/SchedulingUseCase.kt

```
@@ -0,0 +1,123 @@
+                    appendLine("✅ [1단계] RawContents: $timeOfCreatingRawContents s")
+                    appendLine("✅ [2단계] Provisionings: $timeOfCreatingProvisionings s")
+                    appendLine("✅ [3단계] Gens: $timeOfCreatingGens s")
+                    append("-> 전체 : $total s")
+                }
+            }
+
+            applicationEventPublisher.publishEvent(
+                ContentsSchedulingEventDto(
+                    isSuccess = isSuccess,
+                    startTime = startTime,
+                    timeOfCreatingRawContents = String.format("%.3f", timeOfCreatingRawContents),
```

detekt (1.23.8)

[warning] String.format("%.3f", timeOfCreatingRawContents) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)